### ISSUE TITLE
Setup for Actions, Companions, System Settings, etc.

### ISSUE THREAD

**Issue by A-Sparowe** [bdejesus/xiv-bars#29]
I love this tool! My only complaint and thing that would be a great addition to it would be the inclusion of all FFXIV elements that are able to be added to Hotbars. 

What I mean is adding the icons from the player actions menu or any other menu like system configuration, character configuration, etc. 

Since this is built using the FFXIV API, from what I understand anyways, all those icons are already available. 

I also add items like food and potions, stuff like that to my Hotbars but those are things I can live without bein added to the tool. With so many items it would be tie consuming for the developers to include them and I’m sure most users wouldn’t mind not having them included. 

I included a picture of what some of my Hotbars look like. Sorry for the resolution as I’m currently playing on my PS4 and not at a computer to take better screenshots. 

As you can see in the screenshots I use a lot of the system icons and a lot of macros icons added to my Hotbars and it’s becoming really common for a lot of players to use these types of UI setups. Configurations like the “collapsing or adaptive” UI Hotbars are, IMO, commonplace. 

When I start a new job class this tool really helps out a lot. It’s also great for saving references to past setups or sharing setups with friends easily. 

![image](https://user-images.githubusercontent.com/11279897/90969467-b3390880-e4be-11ea-9b60-ce4093c28440.jpg)

![image](https://user-images.githubusercontent.com/11279897/90969515-ff844880-e4be-11ea-91a3-e4438562579c.jpg)

![image](https://user-images.githubusercontent.com/11279897/90969531-1e82da80-e4bf-11ea-9667-917430062e64.jpg)

I’m not saying to include the various ways each Hotbar can be configured, though it would be a nice gesture, but the icons that are in them. Sure some players use different setups other that the default “12x1” row setup but doing the positioning of the icons on the screen is something we, the users can do ourselves.

**Comment by bdejesus** [comment]
Thanks for the feedback! I'm glad that you find this tool useful. 

I have thought about adding all of the available icons/actions to the UI and the challenge is figuring out a way to pull in all this data in an efficient way – currently, these pieces of data are kind of spread out over different endpoints from xivapi. But it is possible – I'll be taking a look at adding these in the short term.

**Comment by bdejesus** [comment]
I've pushed out an update the adds most of the menu icons, macros, mount/pet actions. Items, mounts, and minions will take a little bit more work since they contain multiple pages of data, but I'll create task for it so it'll eventually make it in.

.|.|.|.
--|--|--|--
![image](https://user-images.githubusercontent.com/5114/91629555-c5172180-e97e-11ea-8296-d42810bf6c14.png) | ![image](https://user-images.githubusercontent.com/5114/91629557-cc3e2f80-e97e-11ea-93c1-1f7c72abba08.png) | ![image](https://user-images.githubusercontent.com/5114/91629559-d2cca700-e97e-11ea-83d8-4ca282544dd6.png) | ![image](https://user-images.githubusercontent.com/5114/91629561-d829f180-e97e-11ea-8541-4482a0717628.png)

**Comment by A-Sparowe** [comment]
WOW This is awesome! Man you really went above and beyond! I feel bad for not checking up on my post sooner. I had actually started rounding up all the major icons and did a little research on the types of setups various jobs/classes use because not everyone has the same setup. I figured since I like messing around with photoshop and graphics stuff that I'd lend a hand but man you beat me to it lol!
